Formula/tippecanoe.rb

```diff
@@ -1,8 +1,8 @@
 class Tippecanoe < Formula
   desc "Build vector tilesets from collections of GeoJSON features"
-  homepage "https://github.com/mapbox/tippecanoe"
```

Comment:
you should also probably raise an issue in the original repo to direct folks to your fork.

Reply:
Thanks. We have had some conversations with them, but you're right, I should also do it in pull request form.